documentation/howtos.md

```diff
@@ -54,7 +54,7 @@ ECOMMERCE_REPOS:
     REPO: 'ecommerce.git'
     # Use a branch which contains this fix:
     # https://github.com/open-craft/ecommerce/pull/13/files
-    VERSION: opencraft-release/juniper.2
+    VERSION: opencraft-release/juniper.3
```

Comment:
@toxinu I checked the https://github.com/open-craft/ecommerce fork on OpenCraft, and it doesn't seem to have Juniper 3 yet. Would you prefer to create a branch for it, or would you prefer to keep it as Juniper 2?

Reply:
@nizarmah Right! I will create the `juniper.3` branch for `ecommerce` and cherry-picked the new commits. 👍🏻

Reply:
Ping me for the review :+1: But please add testing instructions because I am not sure how to exactly test the changes.